cmd/pebble/cmd_rm.go

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2022 Canonical Ltd
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License version 3 as
+// published by the Free Software Foundation.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package main
+
+import (
+	"github.com/jessevdk/go-flags"
+
+	"github.com/canonical/pebble/client"
+)
+
+type cmdRm struct {
+	clientMixin
+
+	Recursive bool `short:"r" long:"recursive"`
+
+	Positional struct {
+		Path string `positional-arg-name:"<path>"`
+	} `positional-args:"yes" required:"yes"`
+}
+
+var rmDescs = map[string]string{
+	"recursive": "Create parent directories as needed.",
+}
+
+var shortRmHelp = "Remove files and directories"
+var longRmHelp = `
+The rm command removes a file or directory.
+If --recursive is specified, all files and directories contained
+within the specified path will also be removed.
```

Comment:
IMO the second sentence can be removed, as it should be documented under the `--recursive` argument's help.

Reply:
I agree with you. Here I mimicked the help text of other commands such as `add` which sometimes include a sentence about an option that's relevant to the command.

https://github.com/canonical/pebble/blob/b13464b97acd35b140b1ad2758ce745f0735daf8/cmd/pebble/cmd_add.go#L40-L45